modules/tinymce/src/core/test/ts/browser/SelectionOverridesTest.ts

```diff
@@ -25,20 +25,20 @@ describe('browser.tinymce.core.SelectionOverridesTest', () => {
 
   const assertSelectionIsCaretBlock = (editor: Editor, caretValue: 'before' | 'after') => {
     const selectedNode = editor.selection.getNode();
-    assert.isTrue(isCaretContainerBlock(selectedNode));
+    assert.isTrue(isCaretContainerBlock(selectedNode), 'Selected node should be a fake caret node');
     assert.equal(selectedNode.getAttribute('data-mce-caret'), caretValue);
   };
 
-  const selectBesideContentEditable = (editor: Editor, contentEditableElm: HTMLElement, clickPoint: 'before' | 'after') => {
+  const selectBesideContentEditable = (editor: Editor, contentEditableElm: HTMLElement, clickPoint: 'before' | 'after', offset: number = 8) => {
```

Comment:
Can we keep the comment explaining where the 8 came from? It appears to have been lost in the change

Reply:
There wasn't really anywhere to put it tbh... maybe I should just not make this optional and then I could put the comment where it's used.

Reply:
Have done what I said above. How does that look?

Reply:
LGTM